services/kanto/include/kanto_hilog.h

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2024 Huawei Device Co., Ltd.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef KANTO_HILOG_WRAPPER_H
+#define KANTO_HILOG_WRAPPER_H
+
+#include "hilog/log.h"
+
+namespace OHOS {
+// param of log interface, such as KANTO_HILOGF.
+enum KantoSubModule {
+    KANTO_MODULE_INNERKIT = 0,
+    KANTO_MODULE_CLIENT,
+    KANTO_MODULE_SERVICE,
+    KANTO_MODULE_JAVAKIT, // java kit, defined to avoid repeated use of domain.
+    KANTO_MODULE_JNI,
+    KANTO_MODULE_COMMON,
+    KANTO_MODULE_JS_NAPI,
+    KANTO_MODULE_BUTT,
+};
+
+static constexpr unsigned int BASE_KANTO_DOMAIN_ID = 0xD001C04;
```

Comment:
I am curious how this ID is selected. And how it avoid conflicts with other ID's that end up in the same system, but we do not know about.

Reply:
thanks for pointing it out since the the base id is the same of another service ability. Fixed it now.
As to how it is selected, I think it's just random. Interesting to notice with a `grep` that different parts of the system have the same base domain id:
```
//base/request/request/frameworks/js/napi/include/upload/upload_hilog_wrapper.h:33:static constexpr unsigned int BASE_UPLOAD_DOMAIN_ID = 0xD001C00;
./distributeddatamgr/pasteboard/utils/native/include/pasteboard_hilog.h:35:static constexpr unsigned int BASE_PASTEBOARD_DOMAIN_ID = 0xD001C00;
``` 
I don't know how having conflicting id affects the system.

Reply:
Interesting, still good to have a unique ID for Kanto.